build_package.py

```diff
@@ -1234,6 +1233,31 @@ def create_scriptlets():
     fp.write(
         r"""#!/bin/bash
 
+echo "Check for python."
+# look for 'python' by '/usr/bin/env', exit with error if it is not found.
+/usr/bin/env python --version >/dev/null 2>&1 || \
+{ echo -e >&2 "\e[31mThe suitable python interpreter is not found. Aborting.\e[0m"; \
+echo -e >&2 "The command - '/usr/bin/env python' has to point to the python interpreter with version \e[31m>=2.6 or >=3.5\e[0m."; exit 1; }
+
+# get first two digits from python version.
+version_string=$(/usr/bin/env python --version 2>&1 | grep -Po "\d.\d")
```

Comment:
Since this needs to run on multiple distros, ideally we would be on the safe side and use grep notation which is available in both - gnu grep and bsd grep.

Reply:
Done. No Perl patterns any more :)